test/support/test_config.h

```diff
@@ -106,4 +106,7 @@
 
 #define TEST_HAS_NO_INT128
 
+#define _LIBCPP_MSC_TEST_OF_COMPLEX_POW__NOT_BROKEN (_LIBCPP_VERSION || _MSC_VER)

```

Comment:
Let's remake it as broken and list libstdc++ library specific macro (based on_GLIBCXX_RELEASE)

Our macro naming example: _PSTL_<SCENARIO>_TEST_COMPLEX_POW_BROKEN

Reply:
Please remove one of two `__` (in row) symbols

Reply:
> Please remove one of two `__` (in row) symbols
Fixed.

Reply:
> Let's remake it as broken and list libstdc++ library specific macro (based on_GLIBCXX_RELEASE)
> 
> Our macro naming example: _PSTL__TEST_COMPLEX_POW_BROKEN

Fixed